docs/deploy.rst

```diff
@@ -129,7 +150,7 @@ deployment, exodus-lambda-deploy.yaml, can be run by the `AWS CLI`_ command
         --template-file configuration/exodus-lambda-pkg.yaml \
         --stack-name ... \
         --capabilities ... \
-        --parameter-overrides env=$ENV_TYPE oai=... project=$PROJECT
+        --parameter-overrides env=$ENV_TYPE oai=... project=$PROJECT lambdafunctionrole=...
```

Comment:
I realize it wasn't introduced in this commit but it's odd that 'project' is the only documented parameter given a environment variable. All other parameters are seen with '...' for omission. Either way is fine by me, I'd just prefer they were uniform.

Reply:
I take this back. $ENV and $PROJECT make sense here, as they're defined in the example just before the command.

Reply:
@nathanegillett I removed it, so all three examples share a same style.

Reply:
@nathanegillett Environment variable is used in here `scripts/build-package`, gotta keep this?

Reply:
Yeah, what you have now is fine.